src/lein_monolith/task/info.clj

```diff
@@ -64,6 +64,23 @@
         (dep/select-dependency dep-name coords)))))
 
 
+(defn deps
+  "Print a list of subprojects and the (internal) projects they depend on.
+  Targeting options may be used to scope down the projects listed."
+  [project opts]
+  (let [[monolith subprojects] (u/load-monolith! project)
+        targets (target/select monolith subprojects opts)
+        dependencies (dep/dependency-map subprojects)]
+    (doseq [project-name targets]
+      (doseq [dependency (get dependencies project-name)]
+        (when (if (contains? subprojects dependency)
+                (u/parse-bool (:internal opts "true"))
+                (u/parse-bool (:external opts "false")))
+          (if (:bare opts)
+            (printf "%s\t%s\n" project-name dependency)
```

Comment:
Should allow parameterizing the delimiter?

Reply:
Eh, can do that as a future enhancement, but tabs are pretty universal and work out-of-the-box with tools like `cut`.